fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACFilter.java

```diff
@@ -297,7 +300,12 @@ private boolean hasDeleteClause(final String sparqlString) {
     }
 
     private boolean isSparqlUpdate(final HttpServletRequest request) {
-        return request.getMethod().equals("PATCH") &&
-                contentTypeSPARQLUpdate.equalsIgnoreCase(request.getContentType());
+        try {
+            return request.getMethod().equals("PATCH") &&
+                    request.getContentType() != null && sparqlUpdate.isCompatible(MediaType.valueOf(request
```

Comment:
I think the null check here is redundant since you are also catching `IllegalArgumentException` which is thrown " if the supplied string cannot be parsed or is null."